Resources/Prototypes/Entities/Objects/Fun/russian_revolver.yml

```
@@ -0,0 +1,44 @@
+- type: entity
+  parent: BaseItem
+  id: RussianRevolver
+  name: russian roulette revolver
+  description: A .45 magnum revolver, designed specifically to play Russian Roulette. Useless in combat.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/russianrevolver.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Fun/russianrevolver.rsi
+    size: 10
+  - type: Clothing
+    sprite: Objects/Fun/russianrevolver.rsi
+    quickEquip: false
+    slots:
+    - suitStorage
+    - belt
+  - type: ContainerContainer
+    containers:
+      revolver-ammo: !type:Container
+  - type: RevolverAmmoProvider
+    whitelist:
+      tags:
+        - CartridgeMagnum
+    proto: CartridgeMagnum
+    capacity: 6
+    chambers: [ True, True, True, True, True, True ]
+    ammoSlots: [ null, null, null, null, null, null ]
+    soundEject:
+      path: /Audio/Weapons/Guns/MagOut/revolver_magout.ogg
+    soundInsert:
+      path: /Audio/Weapons/Guns/MagIn/revolver_magin.ogg
```

you don't need to write the default values in the prototype
```suggestion
    chambers: [ True, True, True, True, True, True ]
    ammoSlots: [ null, null, null, null, null, null ]
```